R/zzz.R

```diff
@@ -24,8 +24,10 @@ startup_messages <- function() {
   ))
   if (!skip_version_check) {
     latest_version <- try(suppressWarnings(latest_released_version()), silent = TRUE)
+    current_version <- try(cmdstan_version(), silent = TRUE)
```

Comment:
I think the problem was that the code assumed it could always check the current version number

Reply:
Yep!

devtools::install_github("stan-dev/cmdstanr@fix-274")

works if I clean out `~/.cmdstanr/`

Reply:
Great, works for me too

Reply:
I'll merge this as soon as the checks pass